### PR TITLE
Feature4 branch created for review

### PR DESCRIPTION
**Pull request by SAZZAD1Q2**
Created new Branch feature4 with many changes especially in README file, HTML file, and CSS file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    </section>
+    <!--hero section-->
+    <section class="hero">
+        <h1 class="title">Hey There.<br>I'm Sazzad</h1>
+        <h2 class="sub-title">I am a Softwaer Developer</h2>
+        <p class="detail">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+    </section>
+    <!--social media-->
+    <section class="social-media">
```
Comment (abdulhamiid) [inline index.html:32]: Your design
image

Figma
image

Please kindly remove the white space that is present in your design, that is not present in the Figma template

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!--header section-->
+    <section class="header">
+        <header>
+            <div class="logo"><a>MD SAZZAD</a></div>
+            <nav class="nav-icon">
```
Comment (abdulhamiid) [inline index.html]: As requested by the previous reviewer that we should consider wrapping the text `MD SAZZAD` in an `a` tag. The `<a>` tag defines a hyperlink, which is used to link from one page to another. The most important attribute of the <a> element is the `href` attribute

Please kindly include the href attribute in your anchor tag `<a>MD SAZZAD</a>`
e.g
`<a href="http://example.com">anchored text</a>`

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @SAZZAD1Q2 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, I really appreciate the effort you put in! but there are still some which haven't been addressed yet.

almost-there

## Highlights 🌟

- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## 🏮 🏮IMPORTANT NOTE
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either approved or not valid/invalid unless it is requested otherwise.**


## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (abdulhamiid) [reply index.html:32]: According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) we must stick to the design as much as possible using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                <a href="#"><img src="img/bars/Union.png" alt=""></a>
+                
+            </nav>
+        </header>
+    </section>
+    <!--hero section-->
+    <section class="hero">
+        <h1 class="title">Hey There.<br>I'm Sazzad</h1>
+        <h2 class="sub-title">I am a Softwaer Developer</h2>
```
Comment (pinkmoon25) [inline index.html:27]: - [ ] [opTional] please correct the typo here, Softwaer to Software.

**Review by pinkmoon25 — APPROVED** [review]
Hi @SAZZAD1Q2 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters checks are passing :heavy_check_mark: 
Use GitHub Flow :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SAZZAD1Q2** [comment]
Thank you for approving my project.

**Pull request merged**